department-of-veterans-affairs/va.gov-cms
# VACMS-2502 service mods et

## Description
See #2502 

## Testing done
Visual, behat

## QA steps
As an admin, go to `/node/626/edit` and visually verify:
 - Form widget is now Paragraphs experimental
 - `Service location` is used for add button
 -  Clinic name, Building name/number, Wing, Floor or Room Number are visible
 -  Phone help text reads "Do not add an appointments-specific number, that’s lower on the screen."
 - Hours help text no longer appears
 - Hours caption field no longer appears
 - Hours table only appears when `Use facility hours` is selected
 - `Add row` and `Csv import` no longer appear
